<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATerm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="portComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshBtn">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Baud rate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudComboBox">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectBtn">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="autoScrollCheckBox">
        <property name="text">
         <string>Auto scroll</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendFileBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Send file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearOutputBtn">
        <property name="text">
         <string>Clear output</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="outputTextBrowser">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(236, 236, 236);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="inputLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>EOL:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="EOLComboBox">
        <item>
         <property name="text">
          <string>LF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CR;LF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N/A</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
